packages/middleware-retry/src/retryMiddleware.ts

```diff
@@ -53,7 +53,7 @@ export const retryMiddleware =
           output.$metadata.totalRetryDelay = totalRetryDelay;
           return { response, output };
         } catch (e) {
-          const retryErrorInfo = getRetyErrorInto(e);
+          const retryErrorInfo = getRetyErrorInfo(e);
```

Comment:
nit: typo fix suggestion to push via GitHub review interface
```suggestion
          const retryErrorInfo = getRetryErrorInfo(e);
```